components/deal/IndividualStoreDetail.tsx

```diff
@@ -0,0 +1,234 @@
+            </Box>
+            {/* store address end */}
+
+            {/* store deal count start */}
+            <Box
+              sx={{
+                height: "auto",
+                width: "100%",
+                display: "flex",
+                flexDirection: "row",
+              }}
+            >
+              <LocalOfferOutlinedIcon
+                sx={{ color: theme.palette.primary.main }}
+              />
+              <Typography
+                sx={{ ml: "0.5rem", fontSize: theme.typography.subtitle2.xl }}
+              >{`${individualStoreData?.activeDealsCount} active deals`}</Typography>
+            </Box>
+            {/* store deals count end */}
+
+            {/* store phone number and website start */}
+            <Box
+              gap={4}
+              sx={{
+                height: "auto",
+                width: "100%",
+                display: "flex",
+                flexDirection: "row",
+              }}
+            >
+              <Box
+                component={"div"}
+                sx={{ height: "100%", width: "auto", display: "flex" }}
+              >
+                <CallIcon sx={{ color: theme.palette.primary.main }} />
+                <Typography
+                  sx={{ ml: "0.5rem", fontSize: theme.typography.subtitle2.xl }}
+                >{`${individualStoreData?.phone}`}</Typography>
+              </Box>
+
+              <Box
+                component={"div"}
+                sx={{ height: "100%", width: "auto", display: "flex" }}
+              >
+                <LanguageIcon sx={{ color: theme.palette.primary.main }} />
+                <Typography
+                  sx={{ ml: "0.5rem", fontSize: theme.typography.subtitle2.xl }}
+                >{`${individualStoreData?.website}`}</Typography>
+              </Box>
+            </Box>
+            {/* store phone number and website emd  */}
+          </Box>
+        </Box>
+        {/* title and detail end */}
+
+        {/* store map location start */}
+        {individualStoreData?.address?.zipCode && (
+          <Box
+            component={"div"}
+            sx={{
+              ...all_center,
+              height: "40px",
+              width: "40px",
+              display: "flex",
+              bgcolor: theme.palette.primary.main,
+              alignSelf: "end",
+              ml: "auto",
+              borderRadius: "10px",
+            }}
+          >
+            <SendIcon sx={{ color: theme.palette.common.white }} />
+          </Box>
+        )}
+
+        {/* store map location end */}
+      </Box>
+
+      {/* read more component start  */}
+
+      <Box
+        component={"div"}
+        sx={{
+          height: "auto",
+          width: "100%",
+          display: "flex",
+          flexDirection: "column",
+          mt: "0.5rem",
+        }}
+      >
+        <Typography
+          sx={{ fontSize:theme.typography.subtitle2.xl}}
```

Comment:
Sir in MUI theme inside typography i defined all the required font sizes to related variants so if give font size directly to typography it does not take that variant size, because of this  i give font size like this.